### ISSUE TITLE
Updates fail with recent commit for space check

### ISSUE THREAD

**Issue by PearsonFlyer** [Sonarr/Sonarr#6478]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

The commit referenced below is causing a failure on upgrading on Windows 10. I'm currently seeing this in Prowlarr specifically, but I was told that this should be reported here because it will also cause failures in Sonarr upgrades due to the same issue.

https://github.com/Sonarr/Sonarr/commit/e66ba84fc0b5b120dd4e87f6b8ae1b3c038ee72b

### Expected Behavior

Update occurs without error.

### Steps To Reproduce

Click "Install Update"

### Environment

```markdown
- OS: Windows 10 - I don't know if this occurs in other OSes.
```

### What branch are you running?

Main

### Trace Logs?

https://gist.github.com/PearsonFlyer/993250f02fc77e22a11009890ccf149d

### Anything else?

I'm running Main, so I don't see this issue in Sonarr, but bogdan mentioned that this would be an issue across the board and should be reported for fix here.
